Automattic/wp-calypso
# Staging Sites: Disable creating staging sites if quota threshold exceeded

Related to https://github.com/Automattic/dotcom-forge/issues/1882

## Proposed Changes

In this pr, I propose adding a message to inform the user that his site has exceeded the site quota to create a staging site.

![exceed_quota](https://user-images.githubusercontent.com/497103/226560748-1a970659-1209-459a-b27b-03bb74b63b7a.png)



## Testing Instructions



1. Create a site with a business plan.
2. Navigate to `/hosting-config/<site>`
3. Make the site atomic.
4. Make sure you have exceeded 100GB quota, or fake it ( Automatticians can use their sandbox and change `WPCOM_JSON_API_Media_Storage_Endpoint` class.)
5. Ensure that you see an image like above.
6. Make sure that the button is disabled.

## Pre-merge Checklist
